app/assets/stylesheets/module-planification.scss

```diff
@@ -139,17 +139,29 @@ $white: $color_main_negative;
       text-align: right;
     }
 
+    .flex-20 {
+      flex-basis: 20%;
+    }
+    .flex-10 {
+      flex-basis: 10%;
+    }
+
     .node-title {
       flex-basis: 65%;
       display: flex;
 
       :first-child {
         flex: 0;
       }
+
+      .fa.cursor-pointer {
```

Comment:
While `.cursor-pointer` is within `.node-title` (nothing else gonna be changed), avoid using `.fa` class, since it belongs to FontAwesome